corehq/apps/export/views/list.py

```diff
@@ -517,20 +555,15 @@ def update_emailed_export_data(request, domain):
 
 
 @location_safe
-class DailySavedExportListView(BaseExportListView):
+class DailySavedExportListView(BaseExportListView, DailySavedExportListHelper):
```

Comment:
Did you look into accessing the helper functionality by composition rather than inheritance?  It's strange for these to be used both as a view mixin and as a standalone utility class.

Reply:
Agreed this is unusual, I think I did inheritance because the boundary between what `self` does and what the helper does is a bit blurry (like as you mentioned above, the privilege check is in the helper but one might expect it to be in the view).

I'd like to change all of the class-based views in this file to functional views and have virtually all of the logic contained in the helper, but I ran out of energy. I'll take another look at doing that, I think that'd address a bunch of the slightly weird things about this PR.

Reply:
Went back and looked at changing all the class-based views to functional ones and it's more involved than I remembered - it means making a bunch of updates to the menu code (not surprising) and to other views (which use them to create links between pages within exports). I'm going to call this out of scope right now.